cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,45 @@
 describe('Cypress application', () => {
-  before(() => {
-
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/alerts');
   });
 
   it('should have the ability to assert automatically resolved alerts', () => {
-
+    cy.get('#alertButton').click();
+    cy.on('window:alert', (str) => {
+      expect(str).to.equal('You clicked a button');
+    });
   });
 
   it('should have the ability to assert scheduled allert', () => {
-
+    cy.get('#timerAlertButton').click();
+    cy.on('window:load', (text) => {
```

Comment:
Change 'window:load' event to 'window:alert', since the expectation is for an alert.